@here/harp-materials/lib/EdgeMaterial.ts

```diff
@@ -202,12 +202,22 @@ export class EdgeMaterial extends THREE.RawShaderMaterial
         return this.uniforms.edgeColorMix.value as number;
     }
     set colorMix(value: number) {
+        if (this.uniforms.edgeColorMix.value === value) {
+            return;
+        }
         this.uniforms.edgeColorMix.value = value;
-        this.updateColorMixFeature();
-    }
-
-    private updateColorMixFeature(): void {
-        this.defines.USE_COLOR = this.colorMix > 0.0 ? 1 : 0;
```

Comment:
Here USE_COLOR define was wrongly used, assuming 1 or 0 value, but the shader code makes no assumptions about values but check if USE_COLOR is defined. Probably copy-paste from SolidLineMaterial or somewhere else, so color mix was always applied.